src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -474,19 +474,74 @@ function prepareConstSumQuestions() {
 
         if (!$('#response_submit_button').is(':disabled')
             || isPreview()) {
+            // Add elements for displaying instructions to Constant Sum questions
+            $(`.evalueeLabel-${qnNum}`).each(function () {
+                $(this).before('<p class="text-color-blue align-left text-bold">Note:</p>' +
+                        `<p class="text-color-blue padding-left-35px" id="constSumInstruction-${qnNum}"></p>` +
+                        '<hr class="margin-top-0 border-top-dark-gray">');
+            });
+
+            // Add elements for displaying messages/errors based on user input
+            $(`.evalueeForm-${qnNum}`).each(function (responseIndx) {
+                $(this).after(`<div id="constSumInfo-${qnNum}-${responseIndx}" style="display:none">` +
+                        `<p class="text-color-green padding-left-35px" id="constSumMessage-${qnNum}-${responseIndx}"></p>` +
+                        '<hr class="margin-top-0 border-top-dark-gray">');
+            });
+
             if ($(`#constSumToRecipients-${qnNum}`).val() === 'true') {
                 let numResponses = $(`[name="questionresponsetotal-${qnNum}"]`).val();
                 numResponses = parseInt(numResponses, 10);
 
                 $(`#constSumInfo-${qnNum}-${numResponses - 1}`).show();
+            } else {
+                $(`[id^="constSumInfo-${qnNum}-"]`).show();
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('padding-left-55px');
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('margin-top-15px');
             }
         } else {
             $(`[id^="constSumInfo-${qnNum}-"]`).hide();
         }
     }
 }
+function updateConstSumQnInstructions(qnNum) {
```

Comment:
uhmmm there are a lot of repetition between this function and the `updateConstSumQnMessage`. Is there a way to reduce such repetition?

Reply:
Yes there's a bit of repetition here. Initially the code of this function was part of the message function so there was no repetition. I moved it into another function since the instructions need to be updated only once at the time page is loaded while the messages are updated at every user input. 

Maybe we can use a JS module ? 
const constSumQuestions = (function() {
       // constant declarations
       const ....
      // methods
      var instructions = function () { ... };
      var messages = functions () { ... };

      return {
           instructions: instructions,
           messages: messages
      }
}());

Reply:
uhmmm just see what u can do...the readability is the priority so if after the extraction its not very readable, we can give up on this idea